### PR TITLE
Add AlwaysValidRecaptcha mock and update README.md with example

### PR DESCRIPTION
**Pull request by beberlei**

### CODE REVIEW

File: Form/Type/RecaptchaType.php [hunk 1]
```diff
@@ -8,7 +8,7 @@ use Symfony\Component\Form\FormInterface;
```
Comment (stof) [inline Form/Type/RecaptchaType.php:7]: The permissions should stay to 644 for the files

File: Service/AlwaysValidRecaptcha.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    /**
+     * Valid captcha response?
+     * 
+     * @param string $challenge
+     * @param string $response
+     * @return bool
+     */
+    public function isValid($challenge = null, $response = null)
+    {
+        return true;
```
Comment (stof) [inline Service/AlwaysValidRecaptcha.php:24]: This one is a duplicate of `FunctionalTest\Recaptcha`

**Comment by stof** [comment]
I'm +1 for the interface

File: Form/Type/RecaptchaType.php (hunk 1 above)
Comment (beberlei) [inline Form/Type/RecaptchaType.php:7]: huh? What did i do? :)

File: Service/AlwaysValidRecaptcha.php (hunk 2 above)
Comment (beberlei) [inline Service/AlwaysValidRecaptcha.php:24]: Narf, didnt see that :-)

**Comment by beberlei** [comment]
Closed because of bullshit :-) WE just need the interface.

**Pull request closed without merging**